src/OSPSuite.Core/Domain/Services/IExplicitFormulaParser.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace OSPSuite.Core.Domain.Services
+{
+   public interface IExplicitFormulaParser
+   {
+      string FormulaString { set; get; }
+      double Compute(double[] variableValues, double[] parameterValues);
+      void Parse();
+   }
+
+   public class NullExplicitFormulaParser : IExplicitFormulaParser
```

Comment:
This is the default implementation that does nothing. Required so that we do not reference sImModel in core